Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py

```diff
@@ -210,11 +322,34 @@ def get_details_of_an_abnormal_case_command(client, args):
     return command_results
 
 
-def get_the_latest_threat_intel_feed_command(client, args=None):
+def get_details_of_an_abuse_mailbox_campaign_command(client, args):
+    campaign_id = str(args.get('campaign_id', ''))
+    subtenant = args.get('subtenant', None)
+
+    response = client.get_details_of_an_abuse_mailbox_campaign_request(campaign_id, subtenant)
+    command_results = CommandResults(
+        outputs_prefix='AbnormalSecurity.AbuseCampaign.campaigns',
+        outputs_key_field='campaignId',
+        outputs=response,
+        raw_response=response
+    )
+
+    return command_results
+
+
+def get_employee_identity_analysis_genome_data_command(client, args):
+    email_address = str(args.get('email_address', ''))
+
+    response = client.get_employee_identity_analysis_genome_data_request(email_address)
```

Comment:
Let's add to the response also the email_address (which is unique)
And then update the `outputs_key_field` in the command results with it.